src/components/Edit/edit.scss

```diff
@@ -0,0 +1,33 @@
+p.instructions {
+  font-style: italic;
+  font-size: large;
+}
+
+.cameraButton {
```

Comment:
You don't need this one either. You'll likely rather need the values from the landing.scss file